### ISSUE TITLE
Stochastic RSI (division by zero)

### ISSUE THREAD

**Issue by lukese** [bennycode/trading-signals#378]
I noticed that when the prices are the same for the previous and current candle in StochRSI, there is a Big division by zero issue. After debugging in, it looks like it's because the prev price is subtracting current, therefore the result is 0.

```
if (this.avgGain.isStable) {
    const relativeStrength = this.avgGain.getResult().div(this.avgLoss.getResult());
    return this.setResult(this.maxValue.minus(this.maxValue.div(relativeStrength.add(1))));
}
```

Line 35: node_modules/trading-signals/dist/RSI/RSI.js

**Comment by bennycode** [comment]
**Great catch!** Thanks for reporting it to me. 

I checked what other libraries are doing in this case and saw that defaulting to 100 seems to be reasonable (see [here](https://github.com/Cloud9Trader/TechnicalIndicators/blob/master/rsi.src.js#L85-L86) and [here](https://github.com/anandanand84/technicalindicators/blob/v3.1.0/src/oscillators/RSI.ts#L34-L36)). I made a fix (https://github.com/bennycode/trading-signals/pull/379) and released it with [trading-signals v3.2.1](https://www.npmjs.com/package/trading-signals/v/3.2.1).

Please try out my fix and report back if it helps.

**Comment by lukese** [comment]
Thanks very much, by the way love your work so far.
